### PR TITLE
Tf 01 004 teachworks

### PR DESCRIPTION
**Pull request by jobayeructit**
testimotial added

### CODE REVIEW

File: src/components/TestimonialCarousel/NextArrow.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      h="60px"
+      w="60px"
+      fontSize="xl"
+      color="#000"
+      shadow="2xl"
+      zIndex="3"
+      top="50%"
+      transform="translateY(-50%)"
+      aria-label="Previous"
+    ></IconButton>
```
Comment (saminsss) [inline src/components/TestimonialCarousel/NextArrow.tsx:24]: Does this need ending tag?

File: src/components/TestimonialCarousel/NextArrow.tsx
Comment (saminsss) [inline src/components/TestimonialCarousel/NextArrow.tsx:1]: Move this under components/common (need to create folder if doesn't exist)

File: src/components/TestimonialCarousel/PrevArrow.tsx
Comment (saminsss) [inline src/components/TestimonialCarousel/PrevArrow.tsx:1]: Move this under components/common (need to create folder if doesn't exist)

File: src/components/TestimonialCarousel/NextArrow.tsx
Comment (saminsss) [reply src/components/TestimonialCarousel/NextArrow.tsx:1]: This arrow can be used in multiple places

**Review by saminsss — CHANGES_REQUESTED** [review]

File: src/Pages/Home/HomePage.tsx
Comment (saminsss) [inline src/Pages/Home/HomePage.tsx:1]: Is there a reason why everything was moved from `page.tsx` to `Homepage.tsx`?

File: src/components/TestimonialCarousel/PrevArrow.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      rounded="50%"
+      h="60px"
+      w="60px"
+      fontSize="xl"
+      color="#000"
+      shadow="2xl"
+      zIndex="3"
+      transform="translateY(-50%)"
+      aria-label="Previous"
+    ></IconButton>
```
Comment (saminsss) [inline src/components/TestimonialCarousel/PrevArrow.tsx:24]: Does this need ending tag?

File: src/Pages/Home/HomePage.tsx
Comment (jobayeructit) [reply src/Pages/Home/HomePage.tsx:1]: yes. I created a folder called pages . inside this I made HomePage.tsx and rendered all the homePage components there. there will be multiple pages in our website. currently we are  working on homepage. So I thought If I put all the pages inside page folder that would be more structured.so is that  folder structure ok or I should change?

Comment (saminsss) [reply src/Pages/Home/HomePage.tsx:1]: This is fine

**Pull request closed without merging**